cucumber-expressions/java/src/main/java/io/cucumber/cucumberexpressions/ParameterTypeRegistry.java

```diff
@@ -4,13 +4,8 @@
 
 import java.math.BigDecimal;
 import java.math.BigInteger;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.SortedSet;
-import java.util.TreeSet;
+import java.text.DecimalFormatSymbols;
+import java.util.*;
```

Comment:
I've configured IDEA to not use star imports. Could you do the same?

Reply:
I'll add an enforcer rule in the pom.